projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/wpcom-admin-interface.php

```diff
@@ -116,15 +116,23 @@ function ( $location ) {
 }
 add_filter( 'pre_update_option_wpcom_admin_interface', 'wpcom_admin_interface_pre_update_option', 10, 2 );
 
+const WPCOM_DUPLICATE_VIEW_WITH_TOUR = array(
```

Comment:
I don't think the tour is the differentiating factor here. The "View" switcher is also hidden on all these screens, and it should remain visible on screens like `admin.php?page=stats` or `tools.php?page=advertising`.

Actually, I think adding  `admin.php?page=stats` and `tools.php?page=advertising` to they array of duplicate views was a mistake, because we are not removing the duplicate Calypso screens for those (which is why we don't show the tour and why we don't hide the switcher).

Due to that, I think it'd be less confusing if we have these two arrays:

- `WPCOM_DUPLICATE_VIEW`: Lists of WP Admin paths whose Calypso counterpart has been removed, and therefore we need to:
  - Enforce the WP Admin link in the menu
  - Hide the View switcher
  - Show the tour
- `WPCOM_CLASSIC_INTERFACE_PATHS`: List of WP Admin paths that can be accessed from a enforced WP Admin view that are not enabled for the Default admin interface (e.g. Jetpack Stats, Advertising Tools)
  - Access to these paths should be enabled when visiting a WP Admin view whose Calypso counterpart has been removed

Reply:
@mmtr this PR it's now redundant and replaced by https://github.com/Automattic/jetpack/pull/40690. 

I should've closed it yesterday, but forgot. Sorry!